src/client/modules/my/dataUpload/dataUpload.ts

```diff
@@ -8,9 +8,26 @@ const month = (todaysDate.getMonth() + 1).toString().padStart(2, '0');
 const day = todaysDate.getDate().toString().padStart(2, '0');
 
 export default class DataUploader extends LightningElement {
-    defaultGameId = `${year}${month}${day}a`;
-    errorMessages: string[] = [];
-    showErrors = false;
+    defaultGameId;
+    errorMessages: string[];
+    showErrors;
+
+    constructor(){
+      super();
+      this.errorMessages = [];
+      this.showErrors = false;
+      this.defaultGameId = `${year}${month}${day}a`;
+
+      /*
+      //Uncomment to use, doesn't seem to run the code
```

Comment:
Is this old / should be removed? I don't think we have (or need) a generateId endpoint